### ISSUE TITLE
About fare structure of drt module

### ISSUE THREAD

**Issue by 14841484** [matsim-org/matsim-code-examples#759]
I am researching the impact of sav service using the drt module.
The drt module can rideshare with other travelers, but is it possible to get a discount on the fare when doing so?

The current drt module allows ridesharing, but the fare remains the same as when riding alone.
We believe that a discount would be a more natural fare structure, since there would be no incentive for travelers to rideshare without a fare discount.

**Comment by michalmac** [comment]
Good point. I just added support for custom `DrtFarHandler`s: https://github.com/matsim-org/matsim-libs/pull/2090

Another (more generic) way: do not specify the fare params (in the drt config) and then "manually" add your custom event handler:
```java
addEventHandlerBinding().toInstance(new MyCustomDrtFareHandler(getMode(), params))
```
